governance-exercises/poll-ex1-response.md

```diff
@@ -0,0 +1,46 @@
+- Based on Maker's Governance cycles as detailed [here](https://manual.makerdao.com/governance/governance-cycle), I would assess the suitability of the proposal's polls for either the Weekly or Monthly Governance cycle. Since the proposal author has emphasized the time-sensitivity of the matter, citing their user base's rapid growth trajectory and the need to capitalize on this growth, I would opt to include these polls in the Weekly cycle.
+- Since Bob has already given authorization via forum post for these Spork-related polls, I would then determine how to split these polls so that users are allowed to express their opinions separately.
+- While these actions are closely linked, I would split them into two separate polls to clearly distinguish between the distinct proposed changes.
+  
+  ### Adjust Spork Protocol D3M Parameters and DAI Interest Rate - July 24, 2023:
+  - Increase Spork D3M Debt Ceiling by 50 million from 50 million to 100 million DAI.
+  - Increase Spork D3M Target Available Debt (gap) by 10 million from 5 million to 15 million.
+  - Adjust Spork Protocol DAI Interest Rate to increase the DAI deposit rate by 0.5% from 2% to 2.5%
+  - Increase the Ceiling Increase Cooldown from 28,800 seconds (8 hrs) to 86,400 seconds (24 hrs)
+ 
+  ### Onboard ABC to Spork Protocol - July 24, 2023
+  -  Onboard the token ABC to Spork with the parameters below:
+	- [onboarding parameters for token $ABC]
+
+- After determining the contents of the poll I would choose from the templates folder the most suitable templates to slot this poll content into, which in this case are 'Adjust $protocol_name D3M Parameters - $date_MONTH_DD,_YYYY.md' and 'Onboard $token to Spark Protocol - $date_MONTH_DD,_YYYY.md'.
+- Next, I would publish a response in the forum thread informing the proposers and community of the decision to submit two polls, with rationale included. *See forum response below*
+- I would then begin to draft the polls using the poll templates as noted above. The polls will include a summary of the proposal, voting choices, and links to relevant resources for voters to make informed decisions. I will review the poll drafts against the 'poll creation checklist' to ensure they meet all criteria. Additionally, I will carefully check for any spelling or grammatical errors and make sure the language used is easy to understand.
+- I would then create a draft PR to the repo and request reviews of the polls content from other scope facilitators, the proposers and Bob
+- Using any feedback and notes from reviewers, I will finalise the polls and review them against the 'poll submission checklist', adding all relevant tags based on the potential impact of the proposed changes and their appropriate categories. In this case, relevant tags would include "high-impact", collateral-onboard" and "D3M" before making a PR to push final changes to the main repo branch.
```

Comment:
You would likely want a tag for 'Spork' as well. The tags are there for the benefit of governance, to help them find votes they are interested in. I think it's safely predictable that at some point a governance participant might want to look at all the Spork-related polls.